thinc/loss.py

```diff
@@ -46,11 +46,13 @@ def __init__(
         names: Optional[List[str]] = None,
         missing_value: Optional[Union[str, int]] = None,
         neg_prefix: Optional[str] = None,
+        label_smoothing: Optional[float] = 0.0
```

Comment:
Make sure to always format with `black` before submitting - I'm pretty sure that'll add a `,` at the end of this line.